app/src/main/java/com/myfitnesstracker/dao/IBMI_DAO.kt

```diff
@@ -0,0 +1,2 @@
+interface IBMI_DAO {
```

Comment:
For naming conventions, an interface should always start with an "I"

Reply:
I'd take a different approach here.  Class names should be UpperCamelCase, and should not have underscores.  This particular case is tricky, because it is an acronym, and the rules there are less clear.  Either way, you could name this something like BodyMassIndexDAO, or IBodyMassIndexDAO.